index.d.ts

```diff
@@ -125,7 +127,7 @@ declare module 'enzyme' {
     createRenderer(options: AdapterOptions): Renderer;
     elementToNode(element: ReactElement): RSTNode;
     isValidElement(el: ReactElement): boolean;
-    nodeToElement(node: RSTNode): ReactElement | string;
+    nodeToElement(node: RSTNodeTypes): ReactElement | string;
```

Comment:
So Enzyme actually does call this function with objects that are not `RSTNode`s? Can it be anything from the `rendered` property of an `RSTNode` or is it more limited than that?